projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/components/DeployJob.tsx

```diff
@@ -102,7 +107,10 @@ export async function showCreateDeploymentDialog() {
           });
         }
       } else {
-        await jobRequest('deploy');
+        const success = await jobRequest('deploy');
```

Comment:
nit: I support the idea of having a more complex return type than just string. Please, while doing it, name the consts properly - I'd suggest something like 'isSuccessful' if you decide to return a boolean value for the status.